src/Microsoft.OData.Edm/Vocabularies/Annotations/IEdmVocabularyAnnotationWithDefault.cs

```diff
@@ -0,0 +1,20 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="IEdmVocabularyAnnotation.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+namespace Microsoft.OData.Edm.Vocabularies
+{
+    /// <summary>
+    /// Represents an EDM vocabulary annotation that may use a term's default value as its value.
+    /// </summary>
+    public interface IEdmVocabularyAnnotationWithDefault : IEdmVocabularyAnnotation
+    {
+        /// <summary>
+        /// Gets whether the annotation uses a default value
+        /// (not defined with a provided value).
+        /// </summary>
+        bool UsesDefault { get; }

```

Comment:
>bool UsesDefault { get; } [](http://example.com/codeflow?start=8&length=25)

I still save my opinion that we could not need this interface. And Only use the Value to identify whether it's using default value or not.

Reply:
For backward compatibility, we need the Value property to always return a non-null value (i.e., the value of the annotation, whether it is explicitly specified or taken from the default).  Thus, in order to round-trip, we need the ability to separately track whether the value was explicit or taken from the default.

The setting of this is in the constructor, so the property can be read only. Same is true for the value, term, target etc. -- all of these are specified when we create the annotation and are immutable once created.